electron/src/js/about.ts

```diff
@@ -94,19 +96,23 @@ const showWindow = () => {
     );
 
     // Locales
-    ipcMain.on(EVENT_TYPE.ABOUT.LOCALE_VALUES, (event, labels) => {
-      const isExpected = event.sender.id === aboutWindow.webContents.id;
-      if (isExpected) {
-        const resultLabels = {};
-        labels.forEach(label => (resultLabels[label] = locale.getText(label)));
-        event.sender.send(EVENT_TYPE.ABOUT.LOCALE_RENDER, resultLabels);
+    ipcMain.on(EVENT_TYPE.ABOUT.LOCALE_VALUES, (event: IpcMessageEvent, labels: i18nStringIdentifier[]) => {
+      if (aboutWindow) {
```

Comment:
Why this check? The about window is set above before subscribing to this event if it does not exist.

Reply:
No, it's just set to `undefined` in line 31. If, for whatever reason, `showWindow()` is not run, `aboutWindow` is undefined.

Reply:
The `ipcMain` we are commenting on is defined with in the `showWindow()`. It will be defined after line 47ff

Reply:
As discussed f2f, will move it into `showWindow()`.